curvlinops/kfac.py

```diff
@@ -114,6 +115,14 @@ def __init__(
                 from the parameters. Defaults to ``None``.
             seed: The seed for the random number generator used to draw labels
                 from the model's predictive distribution. Defaults to ``2147483647``.
+            fisher_type: The type of Fisher/GGN to approximate. If ``'type-2'``, the
+                expectation over the model outputs is computed exactly by
+                computing the backwward pass for each output dimension. This is
+                sometimes also called type-2 Fisher. If ``'mc'``, the expectation
+                is approximated by sampling ``mc_samples`` labels from the model's
+                predictive distribution. If ``'empirical'``, the empirical gradients
+                are used which corresponds to the uncentered gradient covariance, or
+                the empirical Fisher. Defaults to ``'mc'``.
```

Comment:
Could mention that `mc_samples` is ignored for empirical and type-2.

Reply:
Added this in the description of `mc_samples` instead of `fisher_type`.